components/Footer.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <footer class="footer">
+    <div class="row">
+      <div class="columns small-12 large-6">
+        <p class="footer__name">Wydział Zarządzania Kryzysowego Urzędu Miasta Katowice</p>
+        <p class="footer__contact">Telefon: <a href="tel: 32 253 95 84">32 253 95 84</a> / E-mail: <a href="mailto: zk@katowice.e">zk@katowice.eu</a></p>
+      </div>
+      <div class="columns small-12 large-6">
+        <ul class="footer__menu">
+          <li v-for="item in menu" :key="item.ID">
+            <nuxt-link class="navigation__element" @click.native="emitClosure" :to="item.slug">{{ item.name }}</nuxt-link>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </footer>
+</template>
+
+<script>
+import API from '../api/connectors/wordpress';
+
+export default {
+  name: 'Footer',
+  data() {
+    return {
+      menu: null
+    }
+  },
+  mounted() {
+    let api = new API(this.$axios);
+    let menu = api.getFooterMenu();
+    menu.then((resp) => {
```

Comment:
Dlaczego to dajemy w mounted? To są dane, które powinny być async czyli albo we vuexie (najlepiej) albo przynajmniej w fetch hooku, o którym pisałem:

https://nuxtjs.org/api/pages-fetch?utm_campaign=Vue.js%20News&utm_medium=email&utm_source=Revue%20newsletter